src/components/spreadsheet/spreadsheet.ts

```diff
@@ -295,8 +295,8 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
     stores.inject(ModelStore, this.model);
     this.notificationStore = useStore(NotificationStore);
     this.composerFocusStore = useStore(ComposerFocusStore);
-    this.sidePanel = useStore(SidePanelStore);
     useStore(ArrayFormulaHighlight);
+    this.sidePanel = useStore(SidePanelStore);
```

Comment:
I'm pretty sure you didn't mean to leave the `ArrayFormulaHighlight` here and move the `SidePanelStore` down one line :p

Reply:
dafuq leftover ><